src/tee_interrupt.rs

```diff
@@ -250,11 +250,11 @@ impl TeeInterruptFunction {
                 vcpu_id: args[1],
                 imsic_mask: args[2],
             }),
-            9 => Ok(TvmCpuRebindImsicEnd {
+            9 => Ok(TvmCpuRebindImsicClone {
                 tvm_id: args[0],
                 vcpu_id: args[1],
             }),
-            10 => Ok(TvmCpuRebindImsicClone {
+			10 => Ok(TvmCpuRebindImsicEnd {
```

Comment:
Indent issue, needs to have `cargo fmt` run.

Reply:
Oops. Sorry I missed it. Fixed now